Scientific_workflows/DEAWaterbodies/DEAWaterbodiesToolkit/ENSOInfluence.ipynb

```diff
@@ -48,11 +48,20 @@
    "outputs": [],
```

Comment:
<p>good - feel free to strengthen this comment if you like eg. by restating that Kati Thanda is wetter during La Niña ?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GeoscienceAustralia/dea-notebooks/pull/724/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='39'/>